spec/support/shared_examples/account_creation.rb

```diff
@@ -9,7 +9,6 @@
         to(include('form-action \'self\' http://localhost:7654'))
     end
 
-    click_submit_default if sp == :saml
```

Comment:
I'm a little confused at needing to remove some of these SAML-specific handling, though as I understand why they were introduced in #6894 to trigger the post-auth redirect that's typically handled in JavaScript, I feel like it would be unexpected that a SAML auth would be "complete" at this point. In `main`, the URL here is the same as what's being asserted below (`UriService.add_params(@saml_authn_request, locale: :es)`), which feels very unexpected. My guess is that based on the timing of the merge of #6894 being after multiple MFA setup was already live in most environments, this was buggy in situations where the config was turned off (as it had been in test environments), and that the changes here are the correct, expected behavior.